src/workspacer.Shared/Workspace/IWorkspaceContainer.cs

```diff
@@ -16,6 +16,8 @@ public interface IWorkspaceContainer
 
         IWorkspace GetNextWorkspace(IWorkspace currentWorkspace);
         IWorkspace GetPreviousWorkspace(IWorkspace currentWorkspace);
+        int GetNextWorkspaceIndex(IWorkspace currentWorkspace);
+        int GetPreviousWorkspaceIndex(IWorkspace currentWorkspace);
 
```

Comment:
Since `GetNextWorkspaceIndex` and `GetPreviousWorkspaceIndex` are being added, I think it would make sense to expose a `GetWorkspaceIndex(IWorkspace workspace)` method too.

Reply:
I agree. I updated the default key bindings and added `GetWorkspaceIndex(IWorkspace workspace)` in 5de1d6c.

Reply:
Awesome! Would you be able to use `GetWorkspaceIndex` in `GetNextWorkspaceIndex` and `GetPreviousWorkspaceIndex` too?